sql/policy_internal.sql

```diff
@@ -65,6 +65,11 @@ BEGIN
     lag := _timescaledb_internal.subtract_integer_from_now(htoid, lag::BIGINT);
   END IF;
 
+  -- chunk status bits:
+  -- 1: compressed
+  -- 2: compressed unordered
```

Comment:
Unordered chunks are not created by INSERTs anymore, but can still be created by UPDATEs, right?

Reply:
We currently dont support UPDATE on compressed chunks. Unordered chunks won't be created by this INSERT change but can still be created by the merge chunk functionality of compression.